### PR TITLE
DR 48 assignments page code refactor

### PR DESCRIPTION
**Pull request by RubySpeeders**
### Summary of Changes

---

### Summary of Implementation Logic

---

### Issue Link

Original Issue: [DR-48](https://www.notion.so/Refactor-Assignments-Page-code-06c91f3694e24f079297a7af41ae303c)

Dependent Issues:

Depends on Issues:

---

### Screen shots / Videos

---

### Acceptance Criteria

---

### Checklist for PR Owner

- [ ] Standardized PR Title
- [ ] Included relevant videos and screen shots
- [ ] Included Summary of Changes
- [ ] Included Summary of Implementation Logic
- [ ] Acceptance Criteria are fufilled

### CODE REVIEW

**Comment by cheesehero112** [comment]
@RubySpeeders The code is working as it should (I see the default image when I chose Jacob), but I got this error message in my terminal. Not sure what's triggering it - do you get this error? 

Screenshot 2024-02-08 at 10 25 30 PM

File: client/src/components/ItemCard.tsx [hunk 1]
```diff
@@ -56,12 +52,7 @@ export const ItemCard = ({ item, onClick }: Props) => {
 
   return (
     <div className={classes.card}>
```
Comment (cheesehero112) [inline client/src/components/ItemCard.tsx:54]: is this supposed to be `classNames={classes.card}` ? I noticed that Line 14 import `import classNames from "classnames";` is not being used

File: client/src/components/ItemCard.tsx [hunk 2]
```diff
@@ -56,12 +52,7 @@ export const ItemCard = ({ item, onClick }: Props) => {
...
-      <div
-        className={classNames(classes.image, {
-          [classes.selected]: item.selected,
-        })}
-        onClick={handleClick}
-      >
+      <div className={classes.image} onClick={handleClick}>
```
Comment (cheesehero112) [inline client/src/components/ItemCard.tsx:55]: same comment as above

File: client/src/app/assignments/page.tsx [hunk 3]
```diff
@@ -66,31 +44,10 @@ export default function Assignments() {
...
-                  src={assignment.instrument.image}
-                  alt={`assigned instrument is ${assignment.instrument.name}`}
-                  width={200}
-                  height={200}
-                />
-              </div>
-              <Typography>{assignment.instrument.name}</Typography>
-            </div>
+            <ItemCard item={assignment.musician} />
+            <ItemCard item={assignment.instrument} />
```
Comment (cheesehero112) [inline client/src/app/assignments/page.tsx:50]: Much shorter & cleaner! Yay, reusable components!!

File: client/src/components/ItemCard.tsx (hunk 1 above)
Comment (RubySpeeders) [reply client/src/components/ItemCard.tsx:51]: ahh! good catch. I deleted the unused import.

**Review by cheesehero112 — APPROVED** [review]
It's working (default image shows up on assignment page) on my end. There might be some conflict with the styling I did in DR-26-Alignment-SelectionPage branch, but I will experiment.

**Pull request merged**